packages/synapse-react-client/.storybook/preview.tsx

```diff
@@ -122,61 +71,111 @@ if (isChromatic() && document.fonts) {
   loaders.push(fontLoader)
 }
 
-export const decorators: Decorator[] = [
-  (Story, context) => {
-    return (
-      <StorybookComponentWrapper storybookContext={context}>
-        <Story />
-      </StorybookComponentWrapper>
-    )
+const preview: Preview = {
```

Comment:
Wrap everything in this `Preview` type, which provides more accurate type-checking